process/smartContract/process.go

```diff
@@ -1144,6 +1213,8 @@ func (sc *scProcessor) processVMOutput(
 	if !createdAsyncCallback && callType == vmcommon.AsynchronousCall {
```

Comment:
if L1213-L1218 were to be extracted to another function, then we could have avoided the else-if situation

Reply:
tried, but does not look good. thanks golang.